data/json/snippets/zombie_anatomy.json

```diff
@@ -0,0 +1,308 @@
+      "<zombie_animal_harvest_bone_general>  <zombie_animal_harvest_general_deeper>"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_humanoid_acid_harvest>",
+    "text": [
+      "<zombie_humanoid_harvest_general>",
+      "<zombie_humanoid_harvest_acid_general>",
+      "<zombie_humanoid_harvest_general>  <zombie_humanoid_harvest_general_deeper>",
+      "<zombie_humanoid_harvest_acid_general>  <zombie_humanoid_harvest_general_deeper>"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_animal_acid_harvest>",
+    "text": [
+      "<zombie_animal_harvest_general>",
+      "<zombie_animal_harvest_acid_general>",
+      "<zombie_animal_harvest_general>  <zombie_animal_harvest_general_deeper>",
+      "<zombie_animal_harvest_acid_general>  <zombie_animal_harvest_general_deeper>"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_humanoid_shadow_harvest>",
+    "text": [
+      "<zombie_humanoid_harvest_general>",
+      "<zombie_humanoid_harvest_shadow_general>",
+      "<zombie_humanoid_harvest_general>  <zombie_humanoid_harvest_general_deeper>",
+      "<zombie_humanoid_harvest_shadow_general>  <zombie_humanoid_harvest_general_deeper>"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_animal_shadow_harvest>",
+    "text": [
+      "<zombie_animal_harvest_general>",
+      "<zombie_animal_harvest_shadow_general>",
+      "<zombie_animal_harvest_general>  <zombie_animal_harvest_general_deeper>",
+      "<zombie_animal_harvest_shadow_general>  <zombie_animal_harvest_general_deeper>"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_hulk_harvest>",
+    "text": [
+      "<zombie_humanoid_harvest_general>",
+      "<zombie_harvest_hulk_general>",
+      "<zombie_humanoid_harvest_general>  <zombie_humanoid_harvest_general_deeper>",
+      "<zombie_harvest_hulk_general>  <zombie_humanoid_harvest_general_deeper>"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_hulk+bone_harvest>",
+    "text": [
+      "<zombie_humanoid_harvest_general>.",
+      "<zombie_harvest_hulk_general>.",
+      "<zombie_humanoid_harvest_bone_general>.",
+      "<zombie_humanoid_harvest_general>.  <zombie_humanoid_harvest_general_deeper>.",
+      "<zombie_harvest_hulk_general>.  <zombie_humanoid_harvest_general_deeper>.",
+      "<zombie_humanoid_harvest_bone_general>.  <zombie_humanoid_harvest_general_deeper>.",
+      "<zombie_harvest_hulk_general>.  <zombie_humanoid_harvest_bone_general>."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_humanoid_kevlar_harvest>",
+    "text": [
+      "<zombie_humanoid_harvest_general>.",
+      "<zombie_harvest_kevlar_general>.",
+      "<zombie_humanoid_harvest_general>.  <zombie_harvest_kevlar_general>.",
+      "<zombie_humanoid_harvest_general_deeper>. <zombie_harvest_kevlar_general>"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "zombie_hulk+kevlar_harvest",
```

Comment:
![Snippets broken](https://github.com/CleverRaven/Cataclysm-DDA/assets/54345792/4e1fa243-52d4-492a-8d2d-1729c90db781)

At least some of your dissection snippets aren't working.

Reply:
I see, those were a last minute addition by Bionicle that I didn't test, apologies.
Looking at the code I see no issue with it in particular? But yes I get the same error as you when doing it.

Reply:
```looks like they forgot to use < > brackets for that one snippet category. you're supposed to be able to reference other snippets that use those brackets from the snippets themselves :blobcatgooglythink:
you can see it done properly here https://github.com/CleverRaven/Cataclysm-DDA/blob/30fe61a8fdc0a46316c719abcc7ca9efe738c3ad/data/json/npcs/talk_tags.json#L522
```

I don't know much about snippet references but maybe this will help?

Reply:
From a skim it looks like just that one is missing <> also please rename it from zombie_hulk+kevlar_harvest to zombie_kevlar_hulk_harvest or zombie_kevlar_2_harvest (to match the monsters id) in both spots, that name is gross

Reply:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/16321433/fc0097a8-371f-4262-9818-ffe04db4a47a)
Yep looks like everything is fixed now, I also caught that I forgot to move leather and fur from zombie generic to zombie animal generic, but that's also fixed now.